### DISCUSSION TITLE
are all oauth 1.0 api's supported by oauth 2.0?

### DISCUSSION THREAD

**Discussion by deadcoder0904** [plhery/node-twitter-api-v2#221] [Q&A] [answered]
is there any place where there's a comparison of which oauth 2.0 methods are supported?

i'm assuming that you can't use oauth 1.0 & oauth 2.0 in conjugation so if 2.0 doesn't have some api available then you can access it via 1.0? is that true?

helps me decide if i should go with the new 2.0 or the old 1.0 :)

**Comment by alkihis** [comment, marked as answer]
OAuth 2 (user context) is restricted to v2 API.

Twitter already have this: https://developer.twitter.com/en/docs/authentication/guides/v2-authentication-mapping
